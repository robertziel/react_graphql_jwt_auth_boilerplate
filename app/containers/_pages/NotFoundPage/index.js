/**
 * NotFoundPage
 *
 * This is the page we show when the user visits a url that doesn't have a route
 *
 */

import React from 'react';
import { FormattedMessage } from 'react-intl';

import { Divider, Grid, H1 } from 'components/_ui-elements';

import messages from './messages';

export default function NotFound() {
  return (
    <Grid container>
      <Grid item xs={12}>
        <H1>
          <FormattedMessage {...messages.header} />
        </H1>
        <Divider />
      </Grid>
    </Grid>
  );
}
