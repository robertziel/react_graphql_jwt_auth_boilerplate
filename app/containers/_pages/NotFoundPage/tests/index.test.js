import React from 'react';
import { IntlProvider } from 'react-intl';

import { mount } from 'enzyme';
import waitForExpect from 'wait-for-expect';

import NotFoundPage from '../Loadable';

function mountWrapper() {
  return mount(
    <IntlProvider locale="en">
      <NotFoundPage />
    </IntlProvider>,
  );
}

let wrapper;

beforeAll(() => {
  wrapper = mountWrapper();
});

describe('<NotFoundPage />', () => {
  it('should render and match the snapshot', async () => {
    await waitForExpect(() => {
      wrapper.update();
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
