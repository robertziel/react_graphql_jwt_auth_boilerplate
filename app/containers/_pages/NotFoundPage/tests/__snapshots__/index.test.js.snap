// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render and match the snapshot 1`] = `"<div class=\\"MuiGrid-root Grid-sc-1hkmrvu-0 NPciN MuiGrid-container\\"><div class=\\"MuiGrid-root Grid-sc-1hkmrvu-0 NPciN MuiGrid-item MuiGrid-grid-xs-12\\"><h1 class=\\"H1-sc-1hnsqhn-0 eVZYlZ\\"><span>404 Not found!</span></h1><hr class=\\"MuiDivider-root Divider-l75xrl-0 gGKxTy\\"></div></div>"`;
